app/assets/stylesheets/partials/_event_box.scss

```diff
@@ -0,0 +1,51 @@
+.event-box {
+    width: 350px;
+    padding: 2.1rem 1.5rem;
+
+    border: 1px solid $v3-color_green-light;
+
+    * { color: $color_font-light !important; }
+
+    &__header {
+        display: flex;
+        align-items: center;
+
+        h2 {
+            margin: 0;
+
+            font-size: $font_size-xlarge;
+        }
+
+        &:after {
+            @extend ._pseudo;
+            @include triangle(50px, 'right', $v3-color_green-light);
+
+            margin-left: 1rem;
+        }
+    }
+
+    ul { margin-top: 1.5rem; }
+
+    li {
+        &:not(:last-of-type) { margin-bottom: 1rem; }
+
+        h4 {
```

Comment:
I am getting a computed font-size of 16px for this h4 element, but the design suggests 22px.
<img width="811" alt="image" src="https://user-images.githubusercontent.com/565647/54715512-05d3ff80-4b2a-11e9-9cbf-de8c2da62fc8.png">

I am ok if we need to adopt to scale it, but the design definitely at least suggests that H4 should be slightly larger than the date and location below it which are 20px.

Reply:
Nice catch. I'll change this to `$font_size-medium`.